feasibility-triangle/troubleshooting/test-flare.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+FLARE_SERVER_URL=${FEASIBILITY_TEST_FLARE_SERVER_URL:-"http://localhost:8084"}
+PRINT_SQ=${FEASIBILITY_TEST_PRINT_SQ:-true}
+CHECK_TRANSLATION=${FEASIBILITY_TEST_CHECK_TRANSLATION:-true}
+CHECK_EXECUTION=${FEASIBILITY_TEST_CHECK_EXECUTION:-true}
+
+json_data=$(cat input-queries.json)
+entries=$(echo "$json_data" | jq -r '.queries[] | @base64')
```

Comment:
With `jq -cM` you should be able to output each query in ndjson format without the need for base64 encoding and decoding later on.

Reply:
@alexanderkiel  - does not seem to work - we should have a look at this together